js/reducers/data-reducer.ts

```diff
@@ -11,7 +11,8 @@ export interface IDataState {
 }
 
 const INITIAL_DATA_STATE = RecordFactory<IDataState>({
-  isFetching: true
+  isFetching: true,
+  error: null
```

Comment:
Is this new error property used anywhere? It doesn't seem to be.

Reply:
I faintly remember that we added this to quickly fix the error reporting code. It should probably be removed, and if the error reporting code breaks then we should fix that code instead of making the change here.

Reply:
@eireland I think is related to the error we were seeing in cypress when the network was not disabled. Looking at this code more closely I now remember that we added this because the `.set("error",...` later on in this file would fail if there wasn't an error property set on the Object.

I think the best thing to do here is to add a test of this error handling, so we can verify this. In the meantime this PR can just leave the code as it was before, so no change from the current state of this PR.